### PR TITLE
fix nsrt learning with cover-multistep-options

### LINKED ISSUE #525: make nsrt_learning with cover_multistep_options work
to reproduce:

`python src/main.py --env cover_multistep_options --approach nsrt_learning --seed 0`

there are at least two issues:
1. data collection times out. this is apparently resolved with `--offline_data_planning_timeout 10`, but we should verify this over many seeds and then commit it to settings.py.
2. the assertions in `_Pick_learned_equivalent_terminal` and `_Place_learned_equivalent_terminal` are broken. they should be turned off at test time, if not removed completely; their purpose was only to validate the _oracle_ samplers.

### PR DESCRIPTION
**Pull request by tomsilver**
closes #525

### CODE REVIEW

File: src/settings.py [hunk 1]
```diff
@@ -188,6 +186,16 @@ def get_arg_specific_settings(args: Dict[str, Any]) -> Dict[str, Any]:
         """A workaround for global settings that are derived from the
         experiment-specific args."""
         return dict(
+            # For learning-based approaches, the data collection timeout for
+            # planning.
+            offline_data_planning_timeout=defaultdict(
```
Comment (ronuchit) [inline src/settings.py:191]: why can't we just make this default across the board? none of our stuff really cares about if training might take a little bit longer right, and usually it'll succeed very fast anyway

File: src/envs/cover.py [hunk 2]
```diff
@@ -1073,18 +1073,8 @@ def _Pick_learned_equivalent_terminal(self, s: State, m: Dict,
                                           o: Sequence[Object],
                                           p: Array) -> bool:
```
Comment (ronuchit) [inline src/envs/cover.py:1074]: how is this function different from Pick_terminal above? i see that `robot` is an extra argument but i'm unsure why

Comment (tomsilver) [reply src/envs/cover.py:1074]: this is related to your question in the other PR. this parameterized option should match the one that we anticipate learning exactly.

**Comment by tomsilver** [comment]
@ronuchit thanks, ready for another look

**Review by ronuchit — APPROVED** [review]

**Pull request merged**